app/Http/Controllers/DashboardController.php

```diff
@@ -15,6 +16,54 @@ public function __construct()
     {
         $this->middleware('auth');
     }
+    public function store(Request $req)
+    {
+        $this->validate($req, [
+            'expAmount' => 'required|numeric',
+            'expType' => 'required|max:255',
+            'expDate' => 'required|date',
+            'expComments' => 'nullable|max:255'
+        ]);
+        $newExpense = new Expense;
+        $newExpense->owner_username= Auth::user()->username;
```

Comment:
Note to @SplitSet3R/box-end you MUST check middleware('auth') in __construct or else we cannot use Auth::user() safely